### DISCUSSION TITLE
Using scalar Long

### DISCUSSION THREAD

**Discussion by sduncan-mbt** [graphql-java/graphql-java#2669] [General]
I'm trying to use scalar Long but it only appears to be avaliable up to version 14. Is this correct?

**Comment by jord1e** [comment]
If you **really** need a Long scalar, you can find it in [graphql-java-extended-scalars](https://github.com/graphql-java/graphql-java-extended-scalars)

RuntimeWiring.newRuntimeWiring()
            .scalar([JavaPrimitives.GraphQLLong](https://github.com/graphql-java/graphql-java-extended-scalars/blob/49138ff098b94b9bee2b87bf6313578d9d11de99/src/main/java/graphql/scalars/java/JavaPrimitives.java#L46))
